nym-vpn-x/src-tauri/src/error.rs

```diff
@@ -20,100 +24,160 @@ pub enum CmdErrorSource {
     Unknown,
 }
 
-#[derive(Error, Debug, Serialize, Deserialize, TS)]
+#[derive(Error, Debug, Serialize, Deserialize, TS, Clone)]
 #[ts(export)]
-pub struct CmdError {
-    #[source]
-    pub source: CmdErrorSource,
+/// Generic error type made to be passed to the frontend and
+/// displayed in the UI as localized error message
+/// `Bkd` stands for 'Backend', source side from where the error is emitted
+pub struct BkdError {
```

Comment:
Wouldn't it be better to name it `BackendError`? You would save a comment + it would be easier to understand the code. Unless, I am unaware of some Rust codestyle...

Reply:
I wanted a short name because this type is used a lot in the codebase, so saving few characters length is a good take.
Please note that use of abbreviation in naming is a usual thing, a few examples that come to mind in rust:
`Vec` `Iter` `Ref` `str` `mpsc` `dbg!` `info` `warn` etc etc

Reply:
I think I agree with @rokas-ambrazevicius here, BackendError seems easier to read while still being relatively short. The types referenced there are not comparable, as they are foundational types that all Rust developers will be familiar with, unlike this type

Reply:
Okay, since at the end this is a matter of personal preferences, I'd like to know @zaneschepke opinion
If you're all afraid of abbreviation naming 😨 I don't mind refactoring `Bkd` to `Backend` ofc

Reply:
My suggestion is based on Swift naming conventions. 
Since Swift and Rust are both object oriented languages, so I think naming conventions between object oriented languages should be quite similar.
In Swift - naming things with long meaningful names, saves a lot of time later on.